packages/injectable/src/dependency-injection-container/createContainer.setuppable.test.js

```diff
@@ -108,23 +118,6 @@ describe('createContainer.setuppable', () => {
     expect(promiseStatus.fulfilled).toBe(false);
   });
 
-  it('given injectable with setup but no way to instantiate, when injected, throws', async () => {
-    const someInjectable = getInjectable({
-      id: 'some-injectable',
-      setup: () => {},
-    });
-
-    const di = getDi(someInjectable);
-
-    await di.runSetups();
-
-    expect(() => {
-      di.inject(someInjectable);
-    }).toThrow(
-      'Tried to inject "some-injectable" when instantiation is not defined.',
```

Comment:
Notice convention of error messages: "Tried to do x, but couldn't because y". Consolidate new errors to convention.